app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,28 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        if (error.status === 429) {
+          this.notify.error(this.l10n.t('Only 5 resend actions are allowed in a minute'));
+        }
+        if (error.status === 422) {
+          this.notify.error(this.l10n.t('Only completed and placed orders can be confirmed'));
```

Comment:
display the actual error message from server, you don't know what caused 422, server does.